### DISCUSSION TITLE
Unable to mount the piccolo admin app to a litestar app.

### DISCUSSION THREAD

**Discussion by amazingakai** [piccolo-orm/piccolo_admin#383] [Q&A]
I'm getting following error mounting piccolo admin router to litestar app.

```py
fastapi: Any = create_admin(tables=TABLE_CONFIGS, site_name="GHSS Soura Admin")

@asgi("/admin-panel/", is_mount=True)
async def admin(scope: "Scope", receive: "Receive", send: "Send") -> None:
    await fastapi(scope, receive, send)
```

```py
Traceback (most recent call last):
  File "/home/akai/Projects/GHSSSoura-Admin/.venv/lib/python3.11/site-packages/uvicorn/protocols/http/h11_impl.py", line 407, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akai/Projects/GHSSSoura-Admin/.venv/lib/python3.11/site-packages/uvicorn/middleware/proxy_headers.py", line 69, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akai/Projects/GHSSSoura-Admin/.venv/lib/python3.11/site-packages/litestar/app.py", line 577, in __call__
    await self.asgi_handler(scope, receive, self._wrap_send(send=send, scope=scope))  # type: ignore[arg-type]
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/akai/Projects/GHSSSoura-Admin/.venv/lib/python3.11/site-packages/litestar/middleware/exceptions/middleware.py", line 222, in __call__
    if litestar_app.logging_config and (logger := litestar_app.logger):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'FastAPI' object has no attribute 'logging_config'
```

**Comment by sinisaos** [comment]
@AmazingAkai I tried the code from [here](https://github.com/sinisaos/simple-piccolo/blob/main/litestar_app.py) and I can't reproduce that error and Piccolo Admin is mounted correctly. Can you post the relevant code here so I can try to reproduce your error?

**Reply by amazingakai** [reply]
Here's relevant code, I only get this error when I open admissions table in the admin panel.

https://gist.github.com/AmazingAkai/4a7af674a5d6ede0eee7cf359629abfe

**Reply by sinisaos** [reply]
@AmazingAkai I've created a new asgi application and I'm using your posted files (a few missing imports, eg `from utils.db import TABLE_CONFIGS`, I don't know what's in that file). As for Piccolo, everything looked good so I started looking in Litestar part. I was unable to get Piccolo admin until I commented [this line](https://gist.github.com/AmazingAkai/4a7af674a5d6ede0eee7cf359629abfe#file-app-py-L70) (Litestar logging middleware). Can you comment out that line and try that? After that everything works fine, at least for me, and admin is displayed. Sorry if it doesn't help, but it's pretty hard to predict exactly what's in your project, so post a link to the github repo next time. As for Piccolo, everything looks good to me.

**Reply by amazingakai** [reply]
Thanks, the issue was because I did some changes to table and forgot to run migrations. Weird error though.
